tests/ui/cast.rs

```diff
@@ -46,12 +46,6 @@ fn main() {
     i64::MAX as u64;
     i128::MAX as u128;
 
-    (-1i8).abs() as u8;
-    (-1i16).abs() as u16;
-    (-1i32).abs() as u32;
-    (-1i64).abs() as u64;
-    (-1isize).abs() as usize;
-
```

Comment:
It would be better not to remove these tests. If we just suppress warnings, we can put `allow(clippy:: cast_abs_to_unsigned)`.

Reply:
Ok, thanks, I was not sure what was the preferred solution.

Both this and msrv should be ok now.